modules/Bio/EnsEMBL/Compara/PipeConfig/Plants/WheatCultivarsProteinTrees_conf.pm

```diff
@@ -70,4 +70,15 @@ sub default_options {
 }
 
 
+sub tweak_analyses {
+    my $self = shift;
+    my $analyses_by_name = shift;
+
+    $analyses_by_name->{'fasttree'}->{'-parameters'}->{'cmd'} = '#fasttree_exe# -nosupport -pseudo -quiet -nopr -wag #alignment_file# > #output_file#';
+
+    # Flow "examl_32_cores" #-2 to "fasttree"
+    $analyses_by_name->{'examl_32_cores'}->{'-flow_into'}{-2} = [ 'fasttree' ];
```

Comment:
I'm surprised this works since everywhere else in our codebase it is written:
```suggestion
    $analyses_by_name->{'examl_32_cores'}->{'-flow_into'}->{-2} = [ 'fasttree' ];
```

Reply:
Every PR review is an opportunity to learn something new 🙂